.github/workflows/build-extension.yml

```diff
@@ -26,10 +26,18 @@ jobs:
         run: yarn lerna run build --scope @stacks/connect-ui
       - name: Build Extension
         run: yarn lerna run prod:ext --stream
+      - name: Copy Extension
+        run: cp -r ./dist ./connect-extension
+        working-directory: packages/app
       - name: Zip extension
-        run: zip -r extension.zip ./packages/app/dist
+        run: zip -r connect-extension.zip ./connect-extension
+        working-directory: packages/app
       - uses: actions/upload-artifact@v2
         name: Upload Zip artifact
         with:
-          name: extension.zip
-          path: extension.zip
\ No newline at end of file
+          name: connect-extension
+          path: packages/app/connect-extension.zip
+      - uses: lucasmotta/pull-request-sticky-header@1.0.0
```

Comment:
great plugin, I got about 30% of the way through building an alternative that fixes at the bottom